opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap.py

```diff
@@ -103,13 +103,11 @@ def _is_installed(req):
 
 def _find_installed_libraries():
     libs = default_instrumentations[:]
-    libs.extend(
-        [
-            v["instrumentation"]
-            for _, v in libraries.items()
-            if _is_installed(v["library"])
-        ]
-    )
+
+    for _, v in libraries.items():
```

Comment:
Is this related to this PR?

Reply:
Maybe we got it by mistake from the previous PR. Reverting to the `main` version of it.